src/web/services/participant.ts

```diff
@@ -185,6 +182,9 @@ export type AddParticipantForm = {
   siteName?: string;
   jobFunction: string;
   crmAgreementNumber: string;
+  firstName: string;
+  lastName: string;
+  email: string;
```

Comment:
Nit: maybe keep them at the top to preserve the order as before since it doesn't look like its in alphabetical order

Reply:
I moved them to the bottom to align with `UpdateParticipantForm` (and the UI itself). I'm not too concerned about preserving order, so I think I'll merge as-is. thanks!